spec/support/vcr.rb

```diff
@@ -35,6 +35,8 @@
   config.before_record do |i|
     i.response.body.force_encoding('UTF-8')
   end
+
+  config.default_cassette_options = { record: ENV.fetch('VCR_RECORD_MODE', :once).to_sym }
```

Comment:
I added this so I didn't have to delete the broken cassette every single time.

Reply:
what to you provide as value, if you want to rerecord? `:once` is the default, so I assume, this is our current mode. If I want a force recreate, what do I put in?